plugins/vendor/microsoft/teams/batch/KUserGraphMetadata.php

```diff
@@ -0,0 +1,169 @@
+<?php
+
+
+class KUserGraphMetadata
+{
+    const CSRF_CIPHER_METHOD = "AES-256-CBC";
+    const MICROSOFT_GRAPH_BEARER_TOKEN = 'microsoftGraphBearerToken';
+    const MICROSOFT_GRAPH_REFRESH_TOKEN = 'microsoftGraphRefreshToken';
+    const MICROSOFT_GRAPH_TOKEN_EXPIRY = 'microsoftGraphTokenExpiry';
+    /**
+     * @var string
+     */
+    public $deltaToken;
+
+    /**
+     * @var string
+     */
+    public $authToken;
+
+    /**
+     * @var string
+     */
+    public $refreshToken;
+
+    /**
+     * @var time
+     */
+    public $authTokenExpiry;
+
+    /**
+     * @var time
+     */
+    public $lastAccessed;
+
+    /**
+     * @var int
+     */
+    public $recordingType;
```

Comment:
recording type here is int but on the previous file it is string - GROUP/PERSONAL